themes/digital.gov/layouts/communities/list.html

```diff
@@ -40,24 +40,16 @@ <h1>{{- .Title | markdownify -}} </h1>
             {{ range $i, $e := $dg_communities }}
 
             <li class="usa-card tablet:grid-col-4">
```

Comment:
If we're going to be using these colors for communities, we should tie them to their colors so it's easier to maintain. You could add a class to each card to differentiate. Then you could look through them in sass.

For example a classes like `cop--fcn` or `cop--federal-communicators-network`. In your SASS you'd have:

```scss
$communities-of-practice: (
  "federal-communicators-network": "#fa9441",
  "multilingual": "#d73a33",
  …
);

@each $community, $color in $communities-of-practice {
  .usa-card.cop--#{community} {
    border-top: #{$color}
  }
}
```

This way you can easily update colors and add/remove communities.

Reply:
Refactored code to the suggested pattern